src/task_1.2/task2.js

```diff
@@ -0,0 +1,28 @@
+const csv = require('csvtojson');
+const { pipeline } = require('stream/promises');
+const path = require('path');
+const fs = require('fs');
```

Comment:
It seems you're bringing the same package to the current file twise.
Consider importing everything you need from the package in one import statement.